public/content/contributing/index.md

```diff
@@ -6,53 +6,53 @@ lang: en
-- [Suggest a project for our DeSci page](/contributing/adding-desci-projects/)
-  _- Add a project built on Ethereum that contributes to decentralized science_
-- [Quizzes](/contributing/quizzes/) _- Add, update, and delete quiz question banks for a relevant page_
-- [Suggest a design resource](/contributing/design/adding-design-resources/) _- Add, update, and delete helpful design resources_
-
-_Any questions?_ 🤔 Reach out on our [Discord server](https://discord.gg/ethereum-org)
+**Translations**
+- [Join the translation program](/contributing/translation-program/) – Help us bring ethereum.org to new languages
+
+**Development**
+- [Work on an open issue](https://github.com/ethereum/ethereum-org-website/issues) – Work we've identified that needs doing
+
+**Design**
+- [Help design the website](/contributing/design/) Designers of all levels can contribute to improve the website
+
+**Content**
+- [Create/edit content](/contributing/#how-to-update-content) – Suggest new pages or make tweaks to what's here already
+- [Add community resources](/contributing/content-resources/) – Add a helpful article or resource to a relevant page
+- [Suggest a design resource](/contributing/design/adding-design-resources/) – Add, update, and delete helpful design resources
+- [Add a glossary term](/contributing/adding-glossary-terms/) – Help us continue to expand the Ethereum [glossary](/glossary/)
+- [Quizzes](/contributing/quizzes/) – Add, update, and delete quiz question banks for a relevant page
+
+**Feature ideas**
+- [Request a feature](https://github.com/ethereum/ethereum-org-website/issues/new?assignees=&labels=Type%3A+Feature&template=feature_request.yaml&title=) – Let us know about any ideas you have for a new feature or design
+
+**Product listings**
+- [Add an exchange](/contributing/adding-exchanges/) – Add an exchange to our [exchange finder](/get-eth/#country-picker)
+- [Add a product](/contributing/adding-products/) – Add a dapp or wallet to a relevant page
+- [Add developer tools](/contributing/adding-developer-tools/) – Add a developer tool to a relevant page
+- [Add a layer 2](/contributing/adding-layer-2s/) – Add a layer 2 to a relevant page
+- [Add a staking product or service](/contributing/adding-staking-products/) – Add a project that helps facilitate solo staking, pooled staking, or staking as a service
+- [Add a wallet](/contributing/adding-wallets/) – Add a wallet for the [find wallets page](/wallets/find-wallet/)
+- [Suggest a project for our DeSci page](/contributing/adding-desci-projects/) – Add a project built on Ethereum that contributes to decentralized science
+
+Any questions? 🤔 Join our [Discord server](https://discord.gg/ethereum-org)
+
+## Good first tasks to start contributing
+
+These are few current tasks that you could help us solve and take responsibility for. For most you will need GitHub account as most changes to the website are made through GitHub.
+
+<IssuesList issues={gfissues} my={8} />
```

Comment:
note: `gfissues` is in the global scope of mdx. Its injected in `<MDXRemote {...mdxSource} components={components} scope={scope} />`